src/core/flow.py

```
@@ -0,0 +1,74 @@
+        self.audio: np.ndarray = audio
+        self.chunks: Optional[np.ndarray] = None
+        self.chunk_starts: Optional[list[float]] = None
+        self.tokens: Optional[np.ndarray] = None
+        self.replicas: Optional[list[Replica]] = None
+
+
+class AudioPreProcessorHandler(BaseTaskHandler):
+    def __init__(self, processor: AudioProcessor):
+        self.processor = processor
+
+    def handle(self, *tasks: AudioTask):
```

Ruff (0.12.2)

Missing return type annotation for public function `handle`

Add return type annotation: `None`

(ANN201)